src/main/java/me/dinowernli/grpc/polyglot/command/Method.java

```diff
@@ -0,0 +1,20 @@
+package me.dinowernli.grpc.polyglot.command;
+
+public class Method {
```

Comment:
Sure, we could hard code the logic to print straight to JSON from the protobuf descriptors. I think though that if there is going to be more than one output format option that it will be much easier to use pre-existing tools to process the java object, eg using gson takes one line to go to JSON, I imagine there will be other options to go to other formats just as easily.